.github/workflows/draft-release.yml

```diff
@@ -8,17 +8,21 @@ on:
         required: true
         default: 'X.Y.Z'
 
+permissions:
+  contents: read
+
 jobs:
   build:
     permissions:
       contents: write
+      secrets: read
```

Comment:
since we are using `github.token` permission to read secrets isn't required.

Reply:
Thanks, fixed.